src/yachtzee.ts

```diff
@@ -1105,6 +1115,21 @@ function prepareOutfitAndFamiliar() {
   }
 }
 
+function stickerSetup(expectedYachts: number) {
+  const currentStickers = $slots`sticker1, sticker2, sticker3`.map((s) => equippedItem(s));
+  const UPC = $item`scratch 'n' sniff UPC sticker`;
+  if (currentStickers.every((sticker) => sticker === UPC)) return;
+  const yachtOpportunityCost = 25 * findLeprechaunMultiplier(bestYachtzeeFamiliar());
+  const embezzlerOpportunityCost = 25 * findLeprechaunMultiplier(meatFamiliar());
+  const value =
+    ((75 - yachtOpportunityCost) * expectedYachts * 2000) / 100 +
+    ((75 - embezzlerOpportunityCost) * Math.min(20, expectedEmbezzlers) * (750 + baseMeat)) / 100;
```

Comment:
I don't like that `value` in `outfit.ts` represents valuePerAdventure, but `value` here represents the entirety of yachtzees and embezzlies we expect to run.

Also, the opportunity costs here can use the structure of the suggested code I commented above.